features/networking/pod.feature

```diff
@@ -190,3 +190,82 @@ Feature: Pod related networking scenarios
     Then the step should succeed
     And the output should not contain "ingress_policing_rate: 1953"
 
+  # @author anusaxen@redhat.com
+  # @case_id OCP-19810
+  @admin
+  Scenario: Conntrack rule for UDP traffic should be removed when the pod for NodePort service deleted
+   Given I store the schedulable nodes in the :nodes clipboard
+   Given I use the "<%= cb.nodes[0].name %>" node
+   And I have a project
+   
+   When I run the :create client command with:
+     | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/pod_with_udp_port_listener.json |
+   Then the step should succeed
+   Given a pod becomes ready with labels:
+     | name=udp-pods |
+   And evaluation of `pod.name` is stored in the :host_pod1_name clipboard
+   And evaluation of `pod.ip` is stored in the :host_pod1_ip clipboard
+   And evaluation of `host_subnet(cb.nodes[0].name).ip` is stored in the :node_ip clipboard
+   
+   When I run the :expose client command with:
+     | resource          | pod |
+     | resource_name     | <%= cb.host_pod1_name %> |
+     | type              | NodePort |
+     | port              | 8080 |
+     | protocol          | UDP |
+   Then the step should succeed
+   
+   When I run the :get client command with:
```

Comment:
it's preferable to use ruby-code and class support instead of calling cli directly.  You can replace line 128-222 by one line like 
`And evaluation of `service(pod.name).node_port(port: 8080)` is stored in the :nodeport clipboard` or just call

Reply:
Thanks Peter for suggestion. Yep seems like i can also do this way. I will check on that